extensions/amp-auto-ads/0.1/wunderkind-network-config.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {Services} from '#service';
+import {buildUrl} from '#ads/google/a4a/shared/url-builder';
+import {dict} from '#core/types/object';
+
+/**
+ * @implements {./ad-network-config.AdNetworkConfigDef}
+ */
+export class WunderkindNetworkConfig {
+  /**
+   * @param {!Element} autoAmpAdsElement
+   */
+  constructor(autoAmpAdsElement) {
+    this.autoAmpAdsElement_ = autoAmpAdsElement;
+  }
+
+  /**
+   * @param {!Window} unused
+   * @override
+   */
+  isEnabled(unused) {
+    return true;
+  }
+
+  /**
+   *
```

Comment:
nit: jsdoc

Reply:
Added

Reply:
I think you might need an `override` here, not sure. You can double check with `amp check-types` if it passes you are all good.